projectq/backends/_sim/_simulator.py

```diff
@@ -97,7 +98,8 @@ def is_available(self, cmd):
         """
         if (cmd.gate == Measure or cmd.gate == Allocate
            or cmd.gate == Deallocate
-           or isinstance(cmd.gate, BasicMathGate)):
+           or isinstance(cmd.gate, BasicMathGate)
+           or isinstance(cmd.gate, TimeEvolution)):
```

Comment:
Do we always want to use the general emulation feature for Hamiltonians with non-commuting terms even if the Hamiltonian has only one term?